src/supermarket/app/controllers/cookbook_versions_controller.rb

```diff
@@ -25,6 +25,7 @@ def show
     @collaborators = @cookbook.collaborators
     @supported_platforms = @version.supported_platforms
     @owner_collaborator = Collaborator.new resourceable: @cookbook, user: @owner
+    @metric_results = @version.metric_results
```

Comment:
Did you happen to play around with `.includes(:metric_results)` in queries? We might start considering that for this and `support_platforms` in this controller and in `CookbooksController` to collect what we need in one pass to the database.